design/ordinal-crm.Rmd

```diff
@@ -0,0 +1,453 @@
+or CLM. The CLM assumes the form: $$
+h[p_{ij}] = h[P(Y_i \leq j)] = \alpha_j + \mathbf x_i^T \boldsymbol \beta, \qquad j = 1, \ldots, c - 1; \, i \geq 1
+$$ where $c \in \{2, 3, \ldots \}$ denotes a number of toxicity categories on ordinal scale, and the
+components of the model are as follows:
+
+1.  $h$ - link function. The most common link functions are "logit" and "probit", which result in
+    Cumulative Logit Model and Cumulative Probit Model respectively.
+2.  $Y_i$ - random variable that represents the response outcome category for subject $i \geq 1$,
+    i.e. $supp(Y_i) = \{1, 2, \ldots, c\},\, i \geq 1$. Probabilities
+    $P(Y_i = j),\, j = 1, \ldots, c$, for $i \geq 1$, are called category probabilities, and the
+    cumulative response probabilities for subject $i \geq 1$ are defined as
+    $$p_{ij} = P(Y_i \leq j) = P(Y_i = 1) + \cdots + P(Y_i = j), \qquad j = 1, \ldots, c; ~ i \geq 1,$$
+3.  $\mathbf x_i$ - a vector of covariates corresponding to main effects and interactions for
+    subject $i \geq 1$.
+4.  $\alpha_j$, $j = 1, \ldots, c - 1$ are "cutpoints" and $\boldsymbol\beta$ is a vector of effects
+    (without intercept term). Although each response category has a corresponding cutpoint, the
+    regression coefficients $\boldsymbol\beta$ are constant across categories. Each cumulative logs
+    has its own intercept ("cutpoint"). Furthermore, if link function $h$ is strictly increasing
+    function of $P(Y_i \leq j)$, $i \geq 1$ (such as logit or probit), then \begin{equation}
+      \label{alpha_ordering0}
+      -\infty < \alpha_1 < \alpha_2 < \cdots < \alpha_{c-1} < \infty,
+    \end{equation} because $P(Y_i \leq j)$ increases in $j$ at any fixed $\mathbf x_i$.
+
+Note that a model for $logit[p_{ij}]$ alone is an ordinary logistic model for a binary response in
+which categories $1$ to $j$ represent "success" and categories $j+1$ to $c$ represent "failure".
+
+### Why cumulative probabilities rather than category probabilities?
+
+As explained by McCullagh and Nelder (1989), the choice and definition of response categories is
+often arbitrary. It is essential, therefore, if we are to arrive at valid conclusions, that the
+nature of those conclusions should not be affected by the number of choice of response categories.
+Such considerations lead to models that based on the cumulative probabilities rather than category
+probabilities. These two sets of probabilities are equivalent, but simple models for the cumulative
+probabilities are likely to have better properties for ordinal response scales than equally simple
+models based on the category probabilities.
+
+### Backward compatible CLM model
+
+In current version of the `crmPack`, we support only two toxicity categories, i.e. no DLT and DLT.
+We code it as 0 (no DLT) and 1 (DLT) and we compute the probability of the DLT, i.e.
+$0 < P(DLT) < 1$. To preserve this schema for multi-category toxicities, we propose the following
+form of the CLM model:
+
+```{=tex}
```

Comment:
this kind of tex stuff should not be needed? just saying begin equation etc. should be enough?

Reply:
This is needed to include the line break after the equation. So it is connected with your previous comment. So, we can have `=tex` wrapping and therefore keep line breaks present.